code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -94,8 +94,8 @@
 	name = "energy sword"
 	desc = "May the Force be with you."
 	icon_state = "sword0"
-	active_force = WEAPON_FORCE_ROBUST
-	active_throwforce = WEAPON_FORCE_ROBUST
+	active_force = 40 // Go forth and slay, padawan
```

Comment:
If you want to use 40, please add 40 to defines and use it here. All of this will probably have to be rebalanced with development of medical system.

Reply:
You could probably use WEAPON_FORCE_LETHAL and reduce its number. It’s currently not used because 61 damage or whatever it is is too much to be used